### ISSUE TITLE
Interaction Issues

### ISSUE THREAD

**Issue by pardeike** [pardeike/Achtung2#50]
I have a multitude of mods in the game, however some of them seem to be restricting me from interacting with the world when using right-click.

Files:
- [Bug - hello hello.txt](https://drive.google.com/file/d/1y1gvJ1fhaLopAxd9mLQ-e1QYn7-DwwDw/view?usp=drivesdk)

-- submitted by caroseknows@gmail.com

**Comment by Woof6968** [comment]
What I had noticed while looking inside the file is I think it's a mod incompatibility with a couple mods that make this happen

**Comment by pardeike** [comment]
Yes, you are confusing the fact that Achtung *reports* the error with it *causing* the error.
